src/pages/documentation/adobe-target/event-reference.md

```diff
@@ -0,0 +1,200 @@
+# Target event reference
+
+<InlineAlert variant="warning" slots="text"/>
+
+This event reference document is applicable to ACPTarget 2.x.
+
+## Events handled
+
+The following events are handled by the Target extension:
+
+### Target content request
+
+This event is created when an API is called to send a Target request for the associated mbox location or to prefetch locations.
```

Comment:
Does "This event is created when an API is called to send a Target or Target prefetch request for the associated mbox location." sound better?